samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/questionpool/QuestionPoolBean.java

```diff
@@ -190,6 +190,10 @@ public QuestionPoolBean()
     resetFields();
   }
 
+  public int getRowIndex() {
+      return qpDataModel.getRowIndex();
```

Comment:
I considering an NPE check in here but I can't see any possibility of qpDataModel being null here so just left this as is